ansible/roles/ooni-bouncer/tasks/main.yml

```diff
@@ -24,9 +24,10 @@
 
 - name: Configure the bouncer file
   template:
-    src: bouncer.yaml.j2
+    src: bouncer.metayaml
```

Comment:
I believe the ansible convention for templates is to make them end in `.j2`. Is there a reason to not follow it in this case?

Reply:
`.j2` makes things actually slightly more painful for me: my editor disables meaningful syntax highlighting for `*.j2` files. E.g. blah.yaml.j2 does not look like yaml file for me. That's why I never used `.j2` suffix. :)